website/app/styles/app.scss

```diff
@@ -74,6 +74,15 @@ body {
       pointer-events: none;
     }
   }
+
+  .hds-flyout[open] {
```

Comment:
@alex-ju I think I've found an OK solution to this (I was hoping to be able to use the existing [`ember-body-class`](https://github.com/mansona/ember-body-class/), that we already use internally, but apparently it doesn't work with dynamic values).

See these two commits:
- https://github.com/hashicorp/design-system/pull/1128/commits/6fd76736354c975177574f1be3e51b8be7787307
- https://github.com/hashicorp/design-system/pull/1128/commits/4065be4468cab150fd00b5d077a668b8e5d53453
Once they're merged you can move "global" modal/flyout specific CSS declaration in their own specific files

Reply:
Ah, cool – I'll follow this pattern then! 🙌

Reply:
I have ported the logic you used in the pagination branch here, for consistency. If the pagination goes in first, I will rebase and drop the commit from this branch.